go/enclave/components/block_processor.go

```diff
@@ -77,9 +82,19 @@ func (bp *l1BlockProcessor) Process(br *common.BlockAndReceipts) (*BlockIngestio
 
 	h := br.Block.Hash()
 	bp.currentL1Head = &h
+	bp.lastIngestedBlock.Mark()
 	return ingestion, nil
 }
 
+func (bp *l1BlockProcessor) HealthCheck() (bool, error) {
```

Comment:
Maybe add a comment to describe what we consider healthy / unhealthy.

Reply:
Good idea 👍